packages/router/README.md

```diff
@@ -40,22 +40,7 @@ The `path` prop specifies the URL path to match, starting with the beginning sla
 
 Some pages should only be visible to authenticated users.
 
-All `Routes` nested in `<Private>` require authentication.
-When a user is not authenticated and attempts to visit this route,
-they will be redirected to the route passed as the `unauthenticated` prop and the originally requested route's path will be added to the querystring in a `redirectTo` param. This lets you send the user to the originally requested once logged in.
-
-```js
-// Routes.js
-<Router>
-  <Route path="/" page={HomePage} name="home" />
-  <Private unauthenticated="home">
-    <Route path="/admin" page={AdminPage} name="admin" />
-  </Private>
-</Router>
-```
-
-Redwood uses the `useAuth` hook under the hood to determine if the user is authenticated.
-Read more about authentication in redwood [here](https://redwoodjs.com/tutorial/authentication).
+We support this using private `<Set>`s or the `<Private>` component. Read more [further down](#private-set)
```

Comment:
- add a period at the end
- also, is it possible to move this up to line 42 so that there's not two paragraphs?
```suggestion
We support this using private `<Set>`s or the `<Private>` component. Read more [further down](#private-set).
```